pydantic/schema.py

```diff
@@ -807,6 +816,20 @@ def field_singleton_schema(  # noqa: C901 (ignore complexity)
     raise ValueError(f'Value not declarable with JSON Schema, field: {field}')
 
 
+def multivalue_literal_field_for_schema(values: Tuple[Any, ...], field: Field) -> Field:
+    field = Field(
```

Comment:
```suggestion
    return Field(
```

Reply:
my dreams of a one-commit pull request dashed

Reply:
sorry. :bowing_man:

I'll try to stop being such a pedant (pun intended).